sdk/communication/communication-phone-numbers/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@azure/communication-phone-numbers",
-  "version": "1.2.0-beta.5",
+  "version": "1.2.0-alpha.20230307.3",
```

Comment:
We should never hardcode an alpha version in the package.json. The alpha versions are set by the pipeline for our daily builds.

Reply:
@weshaggard I was attempting to do an alpha release from that branch to test it out in the portal. if I don't change it manually, it will remain to be the beta version here:

https://dev.azure.com/azure-sdk/internal/_build/results?buildId=2612994&view=artifacts&pathAsName=false&type=publishedArtifacts.  

![image](https://user-images.githubusercontent.com/124081795/225743312-f1b9a098-3522-4fe3-a97f-c375395fb83c.png)

Do you know how to change it version? Thank you!

Reply:
@weshaggard @danielortega-msft Sorry I thought I replied to this message two days ago, it turns out I didn't post the message successfully...

Reply:
@guoqing2023 this isn't the correct way to do an alpha build release. To do that see instructions at https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/107/Dev-Feeds?anchor=manually-queuing-a-dev-build. We should never manually put alpha version in the package.json. Please submit a PR to change this back to beta.1 (or whatever number you are on).